crates/viewer/re_renderer/src/importer/stl.rs

```diff
@@ -20,15 +19,28 @@ pub fn load_stl_from_buffer(
 ) -> Result<CpuModel, StlImportError> {
     re_tracing::profile_function!();
 
-    let cursor = std::io::Cursor::new(buffer);
-    let StlData {
-        name,
-        triangles,
-        normals,
-        ..
-    } = StlData::read_buffer(std::io::BufReader::new(cursor)).map_err(StlImportError::TinyStl)?;
+    let mut cursor = std::io::Cursor::new(buffer);
+    let reader = stl_io::create_stl_reader(&mut cursor).map_err(StlImportError::StlIoError)?;
+    // TODO(hmeyer/stl_io#26): parse name from ASCII?
+    // https://github.com/hmeyer/stl_io/pull/26
+    let name = reader.name().cloned().unwrap_or_default();
```

Comment:
afaik we use this only for debug & error messages. So if we can't get a name we can force callers to provide a debug name.

Comment is confusing now since you do have a name in here due to the patched library version ;-)